README.md

```diff
@@ -127,7 +127,7 @@ source and remove the line that says "#define USE_OPENCL".
 
     # Use stand alone directory to keep source dir clean
     mkdir build && cd build
-    cmake ..
+    cmake -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
Why would people want to run a debug build? Remove this. Not everyone is a developer.

Reply:
The Debug option was added so that assertions would be enabled when running the tests. Without that option, any tests that verified assertions or inadvertently triggered them would fail. Is there a way to have assertions turned on just when running tests? I think we also want them on when doing the CI build. Even though most people are not developers, it might be nice to have assertions/debug on while they run so that if any assertions are hit while running, they will be reported, and whatever problems they indicate, fixed. I'll remove it if you insist, but I at least wanted to offer this opinion.